### ISSUE TITLE
ansible-runner worker --worker-info is returning same UUID for 3 different VM's

### ISSUE THREAD

**Issue by jainnikhil30** [ansible/ansible-runner#850]
`ansible-runner worker --worker-info` is returning the same UUID for three different VM's on IBM Cloud. Looking at the code, it reads the machine-id and puts that as UUID, which is same for all the three vm's on IBM cloud. It seems the machine-id is not a full proof unique value. We should probably construct the unique UUID with something else, may be MAC address ?

The same UUID is causing an issue on Automation Controller installation. These vm's were supposed to be three different execution nodes, but because of the same UUID, that doesnt happen and Automation Controller only shows 1 execution node.

```
TASK [ansible.automation_platform_installer.receptor : Get UUID for execution nodes] ***
^[[0;36mskipping: [165.192.135.199] => {"changed": false, "skip_reason": "Conditional result was False"}^[[0m
^[[0;33mchanged: [158.175.191.143] => {"changed": true, "cmd": "ansible-runner worker --worker-info", "delta": "0:00:00.156578", "end": "2021-09-27 10:34:12.437573", "msg": "", "rc": 0, "start": "2021-09-27 10:34:12.280995", "stderr": "", "stderr_lines": [], "stdout": "{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}", "stdout_lines": ["{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}"]}^[[0m
^[[0;33mchanged: [161.156.198.69] => {"changed": true, "cmd": "ansible-runner worker --worker-info", "delta": "0:00:00.166301", "end": "2021-09-27 10:34:12.484405", "msg": "", "rc": 0, "start": "2021-09-27 10:34:12.318104", "stderr": "", "stderr_lines": [], "stdout": "{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}", "stdout_lines": ["{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}"]}^[[0m
^[[0;33mchanged: [158.175.191.141] => {"changed": true, "cmd": "ansible-runner worker --worker-info", "delta": "0:00:00.160766", "end": "2021-09-27 10:34:12.539002", "msg": "", "rc": 0, "start": "2021-09-27 10:34:12.378236", "stderr": "", "stderr_lines": [], "stdout": "{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}", "stdout_lines": ["{cpu_count: 4, errors: [], mem_in_bytes: 16600657920, runner_version: 2.1.0, uuid: 06dd765ca427f032a6f68d4152c8be32}"]}^[[0m
```

**Comment by Shrews** [comment]
FYI @beeankha @AlanCoding

**Comment by tchellomello** [comment]
@jladdjr

**Comment by jladdjr** [comment]
Interesting. I had wondered if this would happen with containers in general. iirc, confirmed that this was not the case (not remembering which container image I checked though). Clearly, this isn't always the case for vms. I like the suggestion about incorporating the mac address.

**Comment by jainnikhil30** [comment]
@beeankha Tested the fix and it works ! 
```
[root@ansible-tower-devel1-vsi ~]# awx-manage list_instances
[controlplane capacity=137 policy=100%]
	xx.xx.xx.xx capacity=137 version=4.1.0 heartbeat="2021-10-06 05:39:24"

[default capacity=274 policy=100%]
	yy.yy.yy.yy capacity=137 version=ansible-runner-2.1.0 heartbeat="2021-10-06 05:39:24"
	zz.zz.zz.zz capacity=137 version=ansible-runner-2.1.0 heartbeat="2021-10-06 05:39:24
```

It shows all the execution nodes now. Thanks for the quick fix !

**Comment by samdoran** [comment]
Resolved by #854.
